spec/features/courses/fee_or_salary/new_spec.rb

```diff
@@ -57,6 +57,7 @@
     let(:course) do
       c = build(:course, provider: provider)
       c.errors.add(:funding_type, "Invalid")
+      c.errors.add(:program_type, "Invalid")
```

Comment:
If you accept the `.tap` suggestion in `sites/new_spec.rb` below, it's probably worth adding it here too.